wremnants/include/muonCorr.h

```diff
@@ -0,0 +1,30 @@
+#include <eigen3/Eigen/Dense>
+#include <eigen3/unsupported/Eigen/CXX11/Tensor>
+
+double scaleWeight(double weight, double scale) {
+    return std::exp(scale*std::log(std::abs(weight))*std::copysign(1., weight));
+}
+
+template <size_t ETABINS, size_t T>
+Eigen::TensorFixedSize<double, Eigen::Sizes<2, ETABINS>> dummyScaleFromMassWeights(
+    Eigen::TensorFixedSize<double, Eigen::Sizes<T>>& weights, double eta, double scale, bool isW=true) {
+    const double refMass = isW ? 80351.81229 : 91153.50974;
+    const size_t centralIdx = 10;
```

Comment:
Central index is the same for both W and Z?

Reply:
Yes it is, the two extra in Z are the width variations (or sinthetaW, can't remember off the top of my head).